workshop_materials.md

```diff
@@ -0,0 +1,104 @@
+# Workshop Materials
+
+This page is meant to act as a collection of the latest Imaging Applications team workshop materials.
+
+## OMERO-related workshops
+
+### Introduction to OMERO
+
+[OMERO](https://www.openmicroscopy.org/omero/) is an image data management and hosting solution. This workshop covers how to store, view, share, annotate, and analyze image data in OMERO.  
+No prior experience with OMERO is necessary.
+
+[Workshop materials](https://thejacksonlaboratory.github.io/omero-training-materials/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/omero-training-materials)
+
+### [OMERO for Python Users](#omero-for-python-users-1)
+
+## Software tools workshops
+
+### Introduction to CellProfiler
+
+[CellProfiler](https://cellprofiler.org) is a Python-based software package focused on image processing and analysis. This workshop serves as an introduction to the software and covers key concept such as projects, modules, and pipelines, as well as building and testing pipelines. 
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/intro_cellprofiler_workshop/)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/intro_cellprofiler_workshop)
+
+### Introduction to Fiji
+
+[Fiji](https://fiji.sc) is a distribution of the [ImageJ](https://imagej.net/software/imagej/) open-source software package for analyzing scientific images. This workshop covers the basic concepts of image analysis (e.g. pixels, bit-depth) and then key Fiji concepts including the GUI layout, scale bars, measurements, and stacks. Finally, selected analysis applications/workflows are demonstrated, including segmentation, colocalization, and tracking. 
+
+[Workshop slide-deck](https://thejacksonlaboratory.github.io/fiji_workshops/IntroFiji.html)  
+[GitHub repository](https://github.com/TheJacksonLaboratory/fiji_workshops)
+
+### Intermediate Fiji: Macros & Scripting
+`<materials in progress>`
+
+### Introduction to napari
```

Comment:
I put this as a software tool and not python, but maybe should cross list like the Omero for Python?

Reply:
The napari workshop is mostly python, so having it in the Python section and linking from here sounds better in my opinion.